pkg/diff/diff.go

```diff
@@ -53,8 +53,8 @@ const (
 	// Error indicates the resource's management annotation in the API server is invalid.
 	Error = Operation("error")
 
-	// Unmanage indicates the resource's management annotation should be removed from the API Server.
-	Unmanage = Operation("unmanage")
+	// Abandon indicates the resource's management annotation should be removed from the API Server.
```

Comment:
Are we going to distinguish between abandon and unmanage?

Reply:
Is there a difference?
Abandoning is removing it from the inventory and removing the management annotations.

Reply:
If no difference, I'm curious why we need to rename it.

Reply:
Oh, I was just renaming it in this internal enum used by the remediator to be more consistent with the language used by the applier.

Reply:
Gotcha.